src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerIssueApplication.java

```diff
@@ -91,4 +98,18 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .state(result.getState())
             .build();
     }
+
+    public SubmittedCallbackResponse submitted(final CaseDetails<CaseData, State> details,
+                                               final CaseDetails<CaseData, State> beforeDetails) {
+
+        log.info("Caseworker issue application submitted callback invoked for case id: {}", details.getId());
+
+        final Application application = details.getData().getApplication();
+
+        if (application.isSolicitorApplication() && application.isSolicitorServiceMethod()) {
+            ccdUpdateService.submitEvent(details, CASEWORKER_ISSUE_SOLICITOR_SERVICE_PACK);
```

Comment:
worth adding a log here if it satisfies the condition

Reply:
All updated in 
https://github.com/hmcts/nfdiv-case-api/pull/685/commits/c709edd2ed4056ec42f80cbc7c45b87de1c3fe96